src/AzureDevOpsDemoBuilder/Views/Environment/CreateProject.cshtml

```diff
@@ -354,9 +354,9 @@
             </div>
         </div>
     </div>
-    <div class="footer">
-        We are moving to .Net Core 3.0, you are using preview version, log your issues <a class="text-white" href="https://github.com/microsoft/AzureDevOpsDemoGenerator/issues/new" target="_blank">here. We'll be return to main site after couple of hours</a>.
-    </div>
+    <!-- <div class="footer">
```

Comment:
Same as above

Reply:
marked it as Reusable footer for announcement